test-requirements.txt

```diff
@@ -1,9 +1,9 @@
-sphinx!=1.6.6,!=1.6.7,!=2.1.0,!=3.0.0,!=3.4.2,<3 # BSD
+sphinx!=1.6.6,!=1.6.7,!=2.1.0,!=3.0.0,!=3.4.2,>=4.2.0  # BSD
 sphinxcontrib.direct-copy
 sphinxcontrib.readme-to-index
 sphinxcontrib.relative-link-corrector
 sphinx_material
 sphinx-markdown-tables
-sphinx_rtd_theme
+sphinx-rtd-theme>=1.0.0  # MIT
```

Comment:
Shouldn't we use material theme in all documents to keep the consistent look and feel?

Reply:
I just added the right constraints to match  the RTD rules - it could be safely removed as it's not selected in conf.py

Right now nothing works. not a big deal if I do create a second PR.